gamemode/modules/base/cl_jobmodels.lua

```diff
@@ -1,10 +1,31 @@
 -- Create a table for the preferred playermodels
+--
+-- Note: the server column wasn't always there, so players may not have a value
+-- for it, even though the code in this file always adds it. That is why it is
+-- added as a NULLable column.
 sql.Query([[CREATE TABLE IF NOT EXISTS darkp_playermodels(
     jobcmd VARCHAR(45) NOT NULL PRIMARY KEY,
-    model VARCHAR(140) NOT NULL
+    model VARCHAR(140) NOT NULL,
+    server TEXT NULL
 );]])
 
-sql.Query("ALTER TABLE darkp_playermodels ADD COLUMN IF NOT EXISTS server VARCHAR(21);")
```

Comment:
When testing this, I found out that the `IF NOT EXISTS` construct does not exist in SQLite. You have to `PRAGMA table_info(...)` to find out whether the column exists.